UI/DesignWnd.cpp

```diff
@@ -1896,6 +1896,8 @@ void DesignWnd::PartPalette::CompleteConstruction() {
 
     CUIWnd::CompleteConstruction();
 
+    SetMinSize(GG::Pt(PART_CONTROL_WIDTH + 70 ,PART_CONTROL_HEIGHT + 70));
```

Comment:
Where does that magical `70` come from and why is it not `75` or `65`?

Reply:
I borrowed the magic from the options window, (where they used 20/70)

Reply:
Could have a separately defined `const GG::Pt PALETTE_MIN_SIZE{GG::X{PART_CONTROL_WIDTH + 70}, GG::Y{PART_CONTROL_HEIGHT + 70}};` to clarify?